src/dev/impl/DevToys/Strings/en-US/NumberBaseConverter.resw

```diff
@@ -179,4 +179,13 @@
   <data name="SearchKeywords" xml:space="preserve">
     <value>Binary Octal Decimal Hexadecimal</value>
   </data>
+  <data name="AdvancedModeLabel" xml:space="preserve">
```

Comment:
Please apply these changes to every other language (without translating it). This will help translators to know what is to be translated.